torch_geometric/nn/fx.py

```diff
@@ -1,13 +1,17 @@
 import copy
+import warnings
 from typing import Any, Dict, Optional
 
 import torch
-from torch.nn import Module, ModuleDict, ModuleList, Sequential
 
 try:
-    from torch.fx import Graph, GraphModule, Node
+    import torch.fx._symbolic_trace as st
+    from torch.nn import Module, ModuleDict, ModuleList, Sequential
 except (ImportError, ModuleNotFoundError, AttributeError):
     GraphModule, Graph, Node = 'GraphModule', 'Graph', 'Node'
+    st = None
+    warnings.warn("`torch.fx._symbolic_trace` could not be imported:"
```

Comment:
This is a bit nasty. I'd like to require it really but the documentation build failed.

Reply:
What do you mean? Why print the warning here?